packages/monocdk-experiment/build.sh

```diff
@@ -22,7 +24,7 @@ tarball=$PWD/monocdk-experiment-*.tgz
 echo "verifying package..."
 cd $(mktemp -d)
 npm init -y
-npm install ${tarball}
+npm install ${tarball} constructs@${constructs_version}
```

Comment:
Doesn't this run into bootstrapping problems? Is the construct library already published to NPM at the right version here? What IS the versioning strategy for that even?

Reply:
Already published.

Reply:
The versioning strategy is that it has it's own semantic version line. Current version is 1.1.2